tasks/Rocky.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
This file would be better named `setup-Rocky.yml`, but I'd rather make it so the same `setup-RedHat` is run for Rocky, since it's the same base... https://github.com/geerlingguy/ansible-role-mysql/blob/master/tasks/setup-RedHat.yml

Additionally, I would like to make sure rockylinux8 is added to `.github/workflows/CI.yml`.

Reply:
Hi Jeff! Thx!
Sorry, I didn't work with github a lot recently. It seems I can't contribute to someone else's PR?
What is the best approach here - me to create my own fork and PR?